.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch Code Studio",
+      "request": "launch",
+      "type": "chrome",
```

Comment:
Haven't debugged yet, but this isn't working for me on Ubuntu. Probably Chromium vs. Chrome or something.

Reply:
That's exactly it. And unfortunately vscode doesn't have support for like a `launch.local.json` where you can override/extend our config. You can just set your local workspace config which applies to all of your workspaces